<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>config_widget</class>
 <widget class="QWidget" name="config_widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#config_widget{
border-image: url(:/png/config_uinew.jpg);
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QFrame" name="frame">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QPushButton" name="Button_Home">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>660</y>
        <width>110</width>
        <height>110</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/png/homedefault.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="lab_title_man">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>30</y>
        <width>891</width>
        <height>81</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color:#FFFFFF;</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:36pt;&quot;&gt;振 动 台 数 字 自 动 对 中 系 统&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="lab_home">
      <property name="geometry">
       <rect>
        <x>615</x>
        <y>700</y>
        <width>120</width>
        <height>30</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color:#FFFFFF;</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;主界面&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="lab_absp_txt">
      <property name="geometry">
       <rect>
        <x>39</x>
        <y>210</y>
        <width>201</width>
        <height>140</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color:#FFFFFF;</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;当前位置&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="lab_absp_mm">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>210</y>
        <width>72</width>
        <height>140</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color:#FFFFFF;</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;mm&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLCDNumber" name="lcdNum_absp">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>210</y>
        <width>150</width>
        <height>140</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/png/lcd_back.png);</string>
      </property>
     </widget>
     <widget class="QLabel" name="lab_pres_mm">
      <property name="geometry">
       <rect>
        <x>1070</x>
        <y>210</y>
        <width>72</width>
        <height>140</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color:#FFFFFF;</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;KPa&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="lab_pres_txt">
      <property name="geometry">
       <rect>
        <x>669</x>
        <y>210</y>
        <width>201</width>
        <height>140</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color:#FFFFFF;</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;气压值&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLCDNumber" name="lcdNum_pres">
      <property name="geometry">
       <rect>
        <x>890</x>
        <y>210</y>
        <width>150</width>
        <height>140</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/png/lcd_back.png);</string>
      </property>
     </widget>
     <widget class="QLabel" name="lab_title_config">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>368</y>
        <width>1211</width>
        <height>45</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius:5px;color:#FFFFFF;background:rgb(38,104,157);</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:28pt;&quot;&gt;参数设置&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="lab_preposi_mm">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>431</y>
        <width>72</width>
        <height>100</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color:#FFFFFF;</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;mm&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="lab_preposi_txt">
      <property name="geometry">
       <rect>
        <x>39</x>
        <y>431</y>
        <width>201</width>
        <height>100</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color:#FFFFFF;</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;预设位置&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="mylinedit" name="lineEdit_preposi">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>431</y>
        <width>150</width>
        <height>100</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
     <widget class="QLabel" name="lab_freq_hz">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>547</y>
        <width>72</width>
        <height>100</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color:#FFFFFF;</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;Hz&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="lab_freq_txt">
      <property name="geometry">
       <rect>
        <x>39</x>
        <y>547</y>
        <width>201</width>
        <height>100</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color:#FFFFFF;</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;控制频率&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="mylinedit" name="lineEdit_freq">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>547</y>
        <width>150</width>
        <height>100</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
     <widget class="QLabel" name="lab_brightness_percent">
      <property name="geometry">
       <rect>
        <x>1020</x>
        <y>431</y>
        <width>72</width>
        <height>100</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color:#FFFFFF;</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;%&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="lab_brightness_txt">
      <property name="geometry">
       <rect>
        <x>640</x>
        <y>431</y>
        <width>201</width>
        <height>100</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color:#FFFFFF;</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;频幕亮度&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLCDNumber" name="lcdNum_brightness">
      <property name="geometry">
       <rect>
        <x>850</x>
        <y>431</y>
        <width>150</width>
        <height>100</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/png/lcd_back.png);</string>
      </property>
     </widget>
     <widget class="QSlider" name="HlSlider_brightness">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>520</y>
        <width>561</width>
        <height>141</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="Button_logSave">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>671</y>
        <width>90</width>
        <height>90</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/png/log_def.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="lab_logSave">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>700</y>
        <width>191</width>
        <height>30</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color:#FFFFFF;</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;导出日志&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="lab_title_posi">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>150</y>
        <width>1211</width>
        <height>45</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius:5px;color:#FFFFFF;background:rgb(38,104,157);</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:28pt;&quot;&gt;当前状态&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_zylight">
      <property name="geometry">
       <rect>
        <x>1060</x>
        <y>440</y>
        <width>151</width>
        <height>81</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/png/bt_zy.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>mylinedit</class>
   <extends>QLineEdit</extends>
   <header location="global">mylinedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
